content/en/docs/instrumentation/swift/manual.md

```diff
@@ -8,7 +8,7 @@ description: Manual instrumentation for opentelemetry-swift
 ## Setup
 
 The
-[OpenTelemetrySdk](https://github.com/open-telemetry/opentelemetry-swift/blob/main/Sources/OpenTelemetrySdk/OpenTelemetrySdk.swift#L37)
+[OpenTelemetry](https://github.com/open-telemetry/opentelemetry-swift/blob/main/Sources/OpenTelemetryApi/OpenTelemetry.swift#L11)
```

Comment:
@bryce-b, thanks for this PR, but the changes don't make sense linguistically nor technically. I've created #2278 to track the issue.